RxGesture/RxGesture.xcodeproj/project.pbxproj

```diff
@@ -493,7 +493,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
-				FRAMEWORK_SEARCH_PATHS = "../Carthage/Build/**";
+				FRAMEWORK_SEARCH_PATHS = ../Carthage/Build/iOS;
```

Comment:
@eltiren Doesn't it break Carthage on macOS target?

I don't understand because Iv'e been able to use RxGesture with Carthage without this change.